MobileBuy/sample/README.md

```diff
@@ -20,6 +20,23 @@ API_KEY=<your-api-key>
 ANDROID_PAY_PUBLIC_KEY=<your-android-pay-public-key>
 ANDROID_PAY_ENVIRONMENT=com.google.android.gms.wallet.WalletConstants.<ENVIRONMENT_TEST OR ENVIRONMENT_PRODUCTION>
 ```
+## Multi language support
+
+If your store supports multiple languages, Storefront API can return translated resource types and fields.
+Learn more about [translating content](https://help.shopify.com/en/api/guides/multi-language/translating-content-api).
+
+To return translated content, `GraphClient#build` needs to receive a locale as an extra param. For example:
+```
+GraphClient.build(
+  context = mockContext, 
+  shopDomain = shopDomain, 
+  accessToken = accessToken, 
+  configure = {
+    httpClient = okHttpClient
+  }
```

Comment:
Do we need a comma here to separate the `locale` param from `configure`?
```suggestion
  },
```

Reply:
We do. Good catch